containers/layout/Layout.tsx

```diff
@@ -15,13 +15,15 @@ export interface LayoutProps {
   home?: boolean;
   heading?: string;
   title?: string | string[];
+  error: React.ReactNode | null;
```

Comment:
Wouldn't it make sense to pass the error *message* here? We could then check whether it's an empty string (don't display) or not.

We would import the error component, passing it the error message if it gets displayed. Since we don't really change the whole component but only the message, this would make more sense to me. What do you think?

Reply:
That makes sense. It's a good suggestion.
I changed it. So I am passing the error of type `FirebaseError` (this error has the `code` and `message` that I need) instead of passing the error component.